src/renderer/views/Watch/Watch.sass

```diff
@@ -0,0 +1,54 @@
+=dual-column-template
+  grid-template: "video video sidebar" 0fr "info info sidebar" auto "info info sidebar" auto / 1fr 1fr 1fr
+
+=theatre-mode-template
+  grid-template: "video video video" auto "info info sidebar" auto "info info sidebar" auto / 1fr 1fr 1fr
+
+=single-column-template
+  grid-template: "video" auto "info" auto "sidebar" auto / auto
+
+.videoLayout
+  display: grid
+  align-items: start
+  +dual-column-template
+
+  @media only screen and (max-width: 1350px)
+    +theatre-mode-template
+
+  @media only screen and (max-width: 900px)
```

Comment:
I really like the adjustments you did here.  Having the video recommendations on the side regardless if theatre mode is active or not is really nice.  It looks like it doesn't adjust when the window size is shrunk though.  The recommendations should eventually go below the comments section once the window is small enough.  I'll let you decide where that threshold is.

Reply:
They do already go below the comments section! It's at 900px wide, I found that trying to maintain two columns at any smaller than that would make both the description and the recommendations get crunched to the point of being unreadable.

I've removed the max-width: 1500px that was previously present since my monitor is 1440px wide so I couldn't test it. You'll have to make adjustments yourself if you want to keep that.